content/20.pharmaceuticals.md

```diff
@@ -388,12 +398,14 @@ While the reported impact on antibody neutralization needs to be confirmed _in v
 Several strategies have been employed to try to mitigate the risk of diminished antibody neutralization.
 Antibody cocktails such as those already holding an EUA may help overcome the risk for attenuation of the neutralizing activity of a single monoclonal antibody.
 These cocktails consist of antibodies that recognize different epitopes on the spike protein, decreasing the likelihood that a single amino acid change can cause resistance to all antibodies in the cocktail.
+
 However, neutralizing resistance can emerge even against an antibody cocktail if the individual antibodies target subdominant epitopes [@doi:10.1101/2021.02.03.429355].
 Another strategy is to develop broadly neutralizing antibodies that target structures that are highly conserved, as these are less likely to mutate [@doi:10.1016/j.immuni.2007.11.018; @doi:10.1126/science.1171491] or to target epitopes that are insensitive to mutations [@doi:10.1016/j.vaccine.2006.04.054].
-One such antibody (ADG-2) has recently been reported [@doi:10.1126/science.abf4830].
-This antibody targets a highly conserved epitope that overlaps the hACE2 binding site of all clade 1 sarbecoviruses.
+Sotrovimab, one such super-antibody, has been found to be effective against all variants assessed as of August 12, 2021 [@url:https://covdb.stanford.edu/page/susceptibility-data/].
+Another antibody (ADG-2) targets a highly conserved epitope that overlaps the hACE2 binding site of all clade 1 sarbecoviruses [@doi:10.1126/science.abf4830].
 Prophylactic administration of ADG-2 in an immunocompetent mouse model of COVID-19 resulted in protection against viral replication in the lungs and respiratory burden.
-Since the epitope targeted by ADG-2 represents an Achilles’ heel for clade 1 sarbecoviruses, this antibody might be a promising candidate against all circulating variants as well as emerging SARS-related coronaviruses.
+Since the epitope targeted by ADG-2 represents an Achilles’ heel for clade 1 sarbecoviruses, this antibody, like sotorimab, might be a promising candidate against all circulating variants as well as emerging SARS-related coronaviruses.
```

Comment:
```suggestion
Since the epitope targeted by ADG-2 represents an Achilles’ heel for clade 1 sarbecoviruses, this antibody, like sotrovimab, might be a promising candidate against all circulating variants as well as emerging SARS-related coronaviruses.
```